boto3/docs/resource.py

```diff
@@ -118,23 +128,15 @@ def _add_params_description(self, section, identifier_names):
             section.write(f':param {identifier_name}: {description}')
             section.style.new_line()
 
-    def _add_overview_of_members(self, section):
-        for resource_member_type in self.member_map:
-            section.style.new_line()
-            section.write(
-                f'These are the resource\'s available {resource_member_type}:'
-            )
-            section.style.new_line()
-            for member in self.member_map[resource_member_type]:
-                if resource_member_type in (
-                    'attributes',
-                    'collections',
-                    'identifiers',
-                    'references',
-                ):
-                    section.style.li(f':py:attr:`{member}`')
-                else:
-                    section.style.li(f':py:meth:`{member}()`')
+    def _add_overview_of_member_type(self, section, resource_member_type):
+        section.style.new_line()
+        section.write(
+            f'These are the resource\'s available {resource_member_type}:'
```

Comment:
nit; we can fix this in a follow up, but I think this is supposed to be "resources" plural rather than possessive?